.github/workflows/bump-main-loculus-version-to-latest.yaml

```diff
@@ -40,18 +45,18 @@ jobs:
           rm -rf loculus_repo
       
       - name: Create Pull Request
-        uses: peter-evans/create-pull-request@v5
+        uses: peter-evans/create-pull-request@v7
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
-          commit-message: "Update Loculus version to ${{ steps.get_loculus_commit.outputs.LATEST_SHA }}"
-          title: "Update Loculus version to ${{ steps.get_loculus_commit.outputs.LATEST_SHA }}"
+          commit-message: "Update Loculus version to https://github.com/loculus-project/loculus/commit/${{ steps.get_loculus_commit.outputs.LATEST_SHA }}"
+          title: "Update Loculus version to ${{ steps.get_loculus_commit.outputs.LATEST_SHA_SHORT }}"
           body: |
-            ${{ github.actor }} wants to update the Loculus version from ${{ steps.get_current_version.outputs.CURRENT_VERSION }} to ${{ steps.get_loculus_commit.outputs.LATEST_SHA }}.
+            @${{ github.actor }} wants to update the Loculus version from https://github.com/loculus-project/loculus/commit/${{ steps.get_current_version.outputs.CURRENT_VERSION }} to https://github.com/loculus-project/loculus/commit/${{ steps.get_loculus_commit.outputs.LATEST_SHA }}.
```

Comment:
This might break the self-approval check - or did you already glance at that and see that it doesn't? (I haven't looked yet)

Reply:
I'm working on the duplication already :) i had same thought

Reply:
Oh because self approval looks at commit message? this should be in a comment here then in this action that another action relies on the exact text

Reply:
Fix on its way to deduplicate:
<img width="580" alt="iTerm2 2024-09-03 15 55 00" src="https://github.com/user-attachments/assets/00c13e05-7d43-484b-8970-bb93e621302c">

```
git log --pretty=format:"- %s" HEAD~5..HEAD | sed 's/.*#\([0-9]\+\)[^#]*$/- loculus-project\/loculus#\1/'
- Turn hashes into links to loculus prs
- ci: use short loculus sha in branch name, bump actions
- loculus-project/loculus#93
- loculus-project/loculus#84
- loculus-project/loculus#78
```